vsts/publish.sh

```diff
@@ -80,7 +80,7 @@ if [ "$mode" == "internal" ]; then
 else
 
   ## 0. Get artifact filename and commit hash from build
-  prerelease=$(echo $ARTIFACT_PATH/zip/*.zip | rev | cut -d/ -f1 | rev)
+  prerelease=$(echo $ARTIFACT_PATH/Carthage/*.zip | rev | cut -d/ -f1 | rev)
```

Comment:
This is unrelated change. `zip` is a folder name of artifacts that is generated by previous build phase in build pipeline.